frontend/src/MessageAlert.tsx

```diff
@@ -20,8 +26,11 @@ type MessageAlertProps = {
 } & PartialMessageAlertProps
 
 export default function MessageAlert({ message, severity, linkText, href }: MessageAlertProps) {
+  const [contact, setContact] = useState(false)
```

Comment:
To improve readability can we rename this to something like `[showContactMessage, setShowContactMessage]`?